AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -15,31 +15,31 @@ extension ApplePayComponent {
     /// Apple Pay component configuration.
     public struct Configuration {
 
-        /// The context of a current payment. Contains 
-        public let payment: ApplePayPayment
+        /// The context of a current payment. Contains
+        public let applePayPayment: ApplePayPayment
 
         /// The merchant identifier for apple pay.
-        public var merchantIdentifier: String
+        public let merchantIdentifier: String
 
         /// A list of fields that you need for a billing contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public var requiredBillingContactFields: Set<PKContactField> = []
+        public let requiredBillingContactFields: Set<PKContactField>
 
         /// A list of fields that you need for a shipping contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public var requiredShippingContactFields: Set<PKContactField> = []
+        public let requiredShippingContactFields: Set<PKContactField>
 
         /// A pre-populated billing address.
-        public var billingContact: PKContact?
+        public let billingContact: PKContact?
 
         /// The flag to toggle onboarding.
         /// If true, allow the shopper to add cards to Apple Pay if non exists yet.
         /// If false, then Apple Pay is disabled if the shopper doesn't have supported cards on Apple Pay wallet.
-        public var allowOnboarding: Bool
+        public let allowOnboarding: Bool
```

Comment:
I would keep those as vars with default values, to enable merchant to initialize the object and set only the properties they want .

Reply:
For `billingContact` or `requiredShippingContactFields` - this is true.
I can do that in another PR

However, `allowOnboarding` we use on `ApplePayComponent.init?` - it is defining if it is possible or not to create an instance of component.
If we make it `var` - it could be changed at any point of time which could lead to inconsistency.

Reply:
For `billingContact` or `requiredShippingContactFields` - this is true.
I can do that in another PR

However, `allowOnboarding` we use on `ApplePayComponent.init?` - it is defining if it is possible or not to create an instance of component.
Changing this afterward does not make much sense and could lead to mistakes 🤔